src/test/java/com/pontusvision/gdpr/PVTemplateTests.java

```diff
@@ -559,15 +559,15 @@ public void test00007TemplateGetDsarRopaByLawfulBasis() throws InterruptedExcept
 
       String report = new String(Base64.getDecoder().decode(renderReply.getBase64Report().getBytes()));
 
-      String expectedReport = "123-Nome do Usuário, E-mail Corporativo\n" +
-              "CADASTRAMENTO DE CLIENTES PJ-E-mail Corporativo, Nome da Empresa, Telefone, Dados Bancários\n" +
+      String expectedReport = "Prontuário do colaborador-Nome Completo, Data de Nascimento, Local de Nascimento, Estado Civil, RG, CPF, Comprovante de Residência, Certificado de Escolaridade, PIS, Certidão de Casamento, Certidão de Nascimento, Dados Bancários, Raça, Filiação a sindicato, Dados de saúde\n" +
```

Comment:
this value was updated because we added 3 new personal sensitive data: Raça, Filiação a sindicato, Dados de saúde